wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java

```diff
@@ -153,6 +155,26 @@ private static void setRollingBatchSize(Properties lmiProperties) {
                 "option.max_rolling_batch_size", String.valueOf(rollingBatchSize));
     }
 
+    private static void setEntryPoint(
```

Comment:
If there is `model.py` or $model_name.py, we should pick it up, set a `entryPoint` here will disable custom model use case.

Reply:
oh right, if .py file exists in model dir but it's not specified in the option. will fix